### PR TITLE
Contrast 23017

### PR DESCRIPTION
**Pull request by serge17**
Now the plugins adds/appends to an argline property to gradle.properties file during the install task.

### CODE REVIEW

File: src/main/groovy/com/contrastsecurity/VerifyContrast.groovy [hunk 1]
```diff
@@ -114,7 +107,7 @@ class VerifyContrast extends DefaultTask {
         }
 
         for (Application application : applications.applications) {
-            if (applicationName.equals(application.name)) {
+            if (applicationName == application.name) {
```
Comment (gmoore) [inline src/main/groovy/com/contrastsecurity/VerifyContrast.groovy:110]: I thought we had to use .equals here.

**Review by gmoore — CHANGES_REQUESTED** [review]

**Comment by gmoore** [comment]
Can you add some tests? There's a lot fo arg line juggling going on. Will be nice to have it tested.

File: src/main/groovy/com/contrastsecurity/VerifyContrast.groovy (hunk 1 above)
Comment (serge17) [reply src/main/groovy/com/contrastsecurity/VerifyContrast.groovy:110]: Seems like in groovy the preferred way is to use "==" http://grails.asia/groovy-compare-string

Comment (gmoore) [reply src/main/groovy/com/contrastsecurity/VerifyContrast.groovy:110]: Oh joy. Nice job, Groovy.

**Comment by serge17** [comment]
@gmoore Added testBuildArgLine()

**Review by gmoore — APPROVED** [review]

**Pull request merged**